kartothek/core/utils.py

```diff
@@ -24,3 +24,10 @@ def _verify_metadata_version(metadata_version):
 
 def verify_metadata_version(*args, **kwargs):
     return _verify_metadata_version(*args, **kwargs)
+
+
+def _ensure_native_string_type(obj):
```

Comment:
Please rename this to `_ensure_string_type`. The _native_ part was lost a long time agi